pyproject.toml

```
@@ -14,7 +14,7 @@ readme = { file=["README.md", "NEWS"], content-type="text/markdown" }
 
 [project]
 name = "sopel-twitter"
-version = "1.3.9"
+version = "1.3.10"
```

To go along with dropping Sopel 7 (see below), we need a bit more than a patch bump.

```suggestion
version = "1.4.0"
```